server/storage/mvcc/kvstore_test.go

```diff
@@ -603,10 +603,10 @@ func TestHashKVWhenCompacting(t *testing.T) {
 		}
 	}()
 
-	// Compact the store in a goroutine, using RevisionTombstone 9900 to 10000 and close donec when finished
+	// Compact the store in a goroutine, using RevisionTombstone 9950 to 10000 and close donec when finished
 	go func() {
 		defer close(donec)
-		for i := 100; i >= 0; i-- {
+		for i := 50; i >= 0; i-- {
```

Comment:
Curious do we know if there was there any rationale for `100` when this was originally written or was it just an arbitrary number?

I don't see any mention in https://github.com/etcd-io/etcd/commit/8609521ce29fd523d003c1fcb5f943ab9c8247b4 or https://github.com/etcd-io/etcd/pull/8263.

Might just be arbitrary to do that many times but I worry maybe reducing the count is going to mask performance regressions.

Reply:
It should be just an arbitrary number.

Reverted this change as it's already confirmed that it's an environment issue. https://github.com/etcd-io/etcd/issues/17232#issuecomment-1909437223